sdk/consts.go

```diff
@@ -128,12 +129,15 @@ const (
 
 // routes for TransactionService
 const (
-	transactionsRoute                 = "/transaction"
-	transactionRoute                  = "/transaction/%s"
-	transactionStatusRoute            = "/transaction/%s/status"
-	transactionsStatusRoute           = "/transaction/statuses"
-	announceAggregateRoute            = "/transaction/partial"
-	announceAggregateCosignatureRoute = "/transaction/cosignature"
+	transactionsRoute                 = "/transactions"
+	transactionsStatusPluralRoute     = "/transactionStatus"
+	transactionStatusSingularRoute    = "/transactionStatus/%s"
+	confirmedTransactionsRoute        = "/transactions/confirmed"
```

Comment:
You don't need to have 3 different routes for each group of transactions.
You are processing all rotes by the same way, you only are using the different urls(Different part in this url).
We already have example how to solve it.
<img width="1007" alt="image" src="https://user-images.githubusercontent.com/18346821/102200544-a60cb300-3ecd-11eb-972d-7caf0ff415d5.png">